### ISSUE TITLE
[Q] Injecting/embedding cea-608 and/or cta-708

### ISSUE THREAD

**Issue by pkej** [tsduck/tsduck#921]
I've tried to find out if tsduck can embed srt/webvtt as cea-608/cta-708 in streams? I've tried to find any tool that is able to insert it into streams, since that's the only way to create captions that aren't burnt in on many streaming services, not least Workplace from Meta which our org use for company and department-wide meetings, and it seems ffmpeg isn't able to do this.

**Comment by lelegard** [comment]
> I've tried to find out if tsduck can embed srt/webvtt as cea-608/cta-708 in streams?

No, it cannot, sorry. This kind of caption is interspersed inside the video PID. TSDuck does not modify the content of audio or video PID's. Usually, ffmpeg is the preferred solution for that kind of manipulation but if you say it cannot process that kind of caption, I have unfortunately no better idea.
